addons/azureSets/azuresets.lua

```diff
@@ -283,22 +293,20 @@ end
 
 function verify_and_set_spell(id, slot)
     local spell = spells[id]
-    local errorMessage = nil
     if not spell then
-        errorMessage = "spell not found"
+        error("spell not found")
+        return false
     end
     if bluJobLevel and spell.levels and spell.levels[BLU_JOB_ID] and spell.levels[BLU_JOB_ID] > bluJobLevel then
-        errorMessage = "job level too low to set spell"
+        error("job level too low to set spell")
+        return false
     end
     if not have_enough_points_to_add_spell(id) then
-        errorMessage = "cannot set spell, ran out of blue magic points"
+        error("cannot set spell, ran out of blue magic points")
+        return false
     end
     if slot > bluSlots then
-        errorMessage = "slot " .. tostring(slot) .. " unavailable"
-    end
-
-    if errorMessage then
-        error(errorMessage)
+        error("slot " .. tostring(slot) .. " unavailable")
```

Comment:
I saw this was there before, but since you're changing this code, please replace the double quotes with single quotes.